src/components/Review/ReviewSectionRowLastAction.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Grid } from 'semantic-ui-react'
+import getSimplifiedTimeDifference from '../../utils/dateAndTime/getSimplifiedTimeDifference'
+import { ReviewSectionComponentProps } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const ReviewSectionRowLastDate: React.FC<ReviewSectionComponentProps> = ({
```

Comment:
I think this file needs to be renamed to match the component exported

Reply:
true, i renamed to match the functionality: 2096dc3c682fedae4320cc12a66cad709aceab39